scripts/avalanchego-installer.sh

```diff
@@ -363,6 +413,25 @@ create_config_file
 create_service_file
 chmod 644 avalanchego.service
 sudo cp -f avalanchego.service /etc/systemd/system/avalanchego.service
+
+foundIP="$(dig +short myip.opendns.com @resolver1.opendns.com)"
+foundArch="$(uname -m)"                         #get system architecture
+foundOS="$(uname)"                              #get OS
+if [ "$foundOS" != "Linux" ]; then
+  #sorry, don't know you.
+  echo "Unsupported operating system: $foundOS!"
+  echo "Exiting."
+  exit
+fi
+foundIP="$(dig +short myip.opendns.com @resolver1.opendns.com)"
```

Comment:
And once more?

Reply:
this was a copy and paste error. Duplicate block can be deleted.